### ISSUE TITLE
Panda Protection flagged Padlock setup 2.0.0 exe

### ISSUE THREAD

**Issue by SanderDeVeth** [MaKleSoft/padlock#107]
Hi, I noticed today that I couldn't find Padlock on my desktop and I couldn't find it through Windows Start, so I went to your website. I downloaded the setup for the new version, but I wasn't allowed to run it as my Panda Protection flagged it. The report said it contains "W32/exploit.gen". Could you explain why this would be in your program? I can't reach my passwords now, but I'm not going to push through with the installation.

**Comment by MaKleSoft** [comment]
Hm, that is very strange. A quick Google search brings up a couple of other cases where Panda Protection falsely identified a Windows installer as containing the virus. Antivirus programs often use somewhat fuzzy heuristics to detect threads which can lead to false positives. I can only imagine this is one of those cases. Can you attach a copy of the report, please? Meanwhile, I'll try to reproduce this myself.

**Comment by SanderDeVeth** [comment]
Sorry, I used a wrong word I think. It's not so much a report as it just notified me of the danger of running this program and put it in quarantine, so I don't think I have extra information that could help.

**Comment by MaKleSoft** [comment]
So I installed the Panda trial version and am also getting the file reported when I download it. But I don't see any mention of a "W32/exploit.gen". Can you maybe send a screenshot of what you're seeing?

**Comment by SanderDeVeth** [comment]
![image](https://user-images.githubusercontent.com/10462233/29997712-ce460084-901a-11e7-8c02-1a0775a680b1.png)

**Comment by MaKleSoft** [comment]
Thanks! I've submitted a false positive report to Panda support. Let's see what they have to say about this. Just to be save, before you execute the installer, please make sure that the file is signed by "Open Source Developer, Martin Kleinschrodt". You can see the signature in the file properties.

**Comment by SanderDeVeth** [comment]
I removed it from Quarantine, but I still can't install it. I've temporarily disabled my AV, but it didn't help.

**Comment by MaKleSoft** [comment]
How exactly does the installation fail? Are you getting some error message? Maybe try to re-download it? It's possible that Panda did something with the binary that made it unusable. So you're currently running the v1.2.0 beta version? Does that version still work? In our tests, uninstalling the app did not remove the associated support directory on Windows, meaning your data should persist through a reinstall. Just to be save though, better create a backup of your data before doing so.

**Comment by SanderDeVeth** [comment]
I get the "Padlock Setup" screen with the green progress bar. It progresses, resets a couple times, then it displays the size, the amount of files and folders and it just closes. (I used gyazo gif to capture it, it closed really quickly) I've tried downloading it several times now, no difference.
https://i.gyazo.com/98115365b1897f863b56b13abf90cbea.mp4
I'm running the 2.0.0 version according to programs, but I can't find the installation folder.
